com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Drawers.cs

```diff
@@ -1,17 +1,38 @@
 using System.Linq;
 using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
 
 namespace UnityEditor.Rendering.HighDefinition
 {
     using CED = CoreEditorDrawer<SerializedHDCamera>;
 
     static partial class HDCameraUI
     {
-        static readonly ExpandedState<CameraUI.Expandable, Camera> k_ExpandedState = new ExpandedState<CameraUI.Expandable, Camera>(CameraUI.Expandable.Projection, "HDRP");
+        /// <summary>Enum to store know the expanded state of a expandable section on the camera inspector</summary>
+        [HDRPHelpURL("HDRP-Camera")]
+        public enum Expandable
```

Comment:
This continue to move \o/ ( Hope it will finally stay in place in the future :D )

Reply:
Lights have this change, materials are coming in another PR too(only URP for the moment)